UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneExporter.cs

```diff
@@ -4,17 +4,38 @@
 using GLTF.Schema;
 using UnityEngine;
 using UnityGLTF.Extensions;
+using UnityEditor;
 
 namespace UnityGLTF
 {
 	public class GLTFSceneExporter
 	{
+        enum TextureMapType
+        {
+			//sRGB
+            Main,
```

Comment:
You've got some odd spacing issues in your changes. I think this project uses tabs.

Reply:
That's weird, I'll fix that!